react/features/notifications/actions.js

```diff
@@ -81,9 +83,12 @@ export function showErrorNotification(props: Object) {
 export function showNotification(props: Object = {}, timeout: ?number) {
     return function(dispatch: Function, getState: Function) {
         const { notifications } = getState()['features/base/config'];
-        const shouldDisplay = !notifications
-            || notifications.includes(props.descriptionKey)
-            || notifications.includes(props.titleKey);
+        const enabledFlag = getFeatureFlag(getState(), NOTIFICATIONS_ENABLED, true);
```

Comment:
please check if this breaks the web, since it has no notion of feature flags

Reply:
I've checked and on web it just defaults to true.